tests/observability/conftest.py

```diff
@@ -112,3 +112,16 @@ def initial_virt_operator_replicas_reverted(prometheus, initial_virt_operator_re
     validate_initial_virt_operator_replicas_reverted(
         prometheus=prometheus, initial_virt_operator_replicas=initial_virt_operator_replicas
     )
+
+
+@pytest.fixture()
```

Comment:
if you need the VM before/during/after the upgrade - the fixture scope should be "session"